src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureValueController.php

```diff
@@ -48,14 +56,115 @@ public function indexAction(Request $request, FeatureValueFilters $filters): Res
             'enableSidebar' => true,
             'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
             'featureValueGrid' => $this->presentGrid($featureValueGridFactory->getGrid($filters)),
-            // @todo: uncomment when add action is migrated
-            //            'layoutHeaderToolbarBtn' => [
-            //                'add_feature_value' => [
-            //                    'href' => $this->generateUrl('admin_feature_values_add', ['featureId' => $featureId]),
-            //                    'desc' => $this->trans('Add new feature value', 'Admin.Catalog.Feature'),
-            //                    'icon' => 'add_circle_outline',
-            //                ],
-            //            ],
+            'layoutHeaderToolbarBtn' => [
+                'add_feature_value' => [
+                    'href' => $this->generateUrl('admin_feature_values_add', ['featureId' => $filters->getFeatureId()]),
+                    'desc' => $this->trans('Add new feature value', 'Admin.Catalog.Feature'),
```

Comment:
Suggestion for improvement in another PR, we could get the parent feature name and adapt the label `Add a feature value to "Material"` (and similar in the creation form title)

Reply:
could be done, but I don't see much value in it. Especially when the behavior allows you to choose another feature when you are in creation page